code/modules/atmospherics/machinery/components/trinary_devices/filter.dm

```diff
@@ -12,6 +12,14 @@
 /obj/machinery/atmospherics/components/trinary/filter/flipped
 	icon_state = "filter_off_f"
 	flipped = 1
+	
+// These two filter types have critical_machine flagged to on and thus causes the area they are in to be exempt from the Grid Check event.
+	
+/obj/machinery/atmospherics/components/trinary/filter/critical
+	critical_machine = 1
```

Comment:
@lzimann I'm happy to change it to True but by the default it shows as a 0. Will TRUE be a problem?